src/decode-tcp.c

```diff
@@ -47,12 +47,12 @@
     (dst).len  = (src).len; \
     (dst).data = (src).data
 
-static int DecodeTCPOptions(Packet *p, uint8_t *pkt, uint16_t len)
+static int DecodeTCPOptions(Packet *p, uint8_t *pkt, uint32_t len)
```

Comment:
not sure I understand why TCP options could be >64k?

Reply:
Again, just for standard-enforcing. I'm ok with reversing the change to uint16